ignite/contrib/handlers/lr_finder.py

```diff
@@ -76,7 +76,7 @@ def __init__(self) -> None:
         self._history = {}  # type: Dict[str, List[Any]]
         self._best_loss = None
         self._lr_schedule = None  # type: Optional[Union[LRScheduler, PiecewiseLinear]]
-        self.logger = logging.getLogger(__name__)
+        self.logger = logging.getLogger(__name__ + "." + self.__class__.__name__)
```

Comment:
Should we do something here with the logger @sdesrozis ?

Reply:
The logger has a name according to the handler (i.e. not root logger), it can be used in the `Parallel` context and be overridden by users. It looks perfect.